SoS/SoS.tp2

```diff
@@ -837,6 +814,20 @@ PRINT @20409
 
 INCLUDE ~%MOD_FOLDER%/Lib/install-itm.tpa~
 
+// wizard spell items
```

Comment:
Can those items be installed without SoS' Wizard spells?

Reply:
Both items use the CBWI429.spl.
The CBRJR008.itm item is the scroll to learn the spell. This also works without an entry in the spell.ids.
In the SoS wizard spell component, the CBWI429 becomes the SPWI429 spell and the tems get the new entry